[Security] 8.18.2 release notes (#6853)
elastic/security-docs: docs/release-notes/8.18.asciidoc

[[release-notes-header-8.18.0]]
== 8.18

[discrete]
[[release-notes-8.18.2]]
=== 8.18.2

[discrete]
[[bug-fixes-8.18.2]]
==== Fixes

> * Improves error telemetry for the AI assistant  ({kibana-pull}220938[#220938]).

Reviewer: ```suggestion
* Improves error telemetry for the AI assistant ({kibana-pull}220938[#220938]).
```

* Simplifies and improves the rule import error message ({kibana-pull}218701[#218701]).

> * Improves the dashboards overview page by moving visualization responses to visualization tables ({kibana-pull}214888[#214888]).

Reviewer 2: The PR describes the larger internal technical improvement that besides other things fixed the issue that's linked in the PR description

https://github.com/elastic/kibana/issues/214733

So the gist of the change is

`Fix Dashboard overview page crash on lens widget aggregation error`

Author: Thanks! For future reference, the PR summary is [pulled directly](https://docs.elastic.dev/platform-docs/kibana-release-notes-best-practices) from the PR title and I usually revise it a bit after briefly scanning the PR description. If there's a specific summary that you'd prefer be used, putting it in the PR title is the best way to ensure it's picked up. Alternatively, you can add the summary in a release notes section to the PR description, similar to what the Endpoint team does ([example](https://github.com/elastic/endpoint-dev/pull/16300)).

Author: Revised in https://github.com/elastic/security-docs/pull/6853/commits/6f04b4879e28d8bcb75ff0dad7a0f8f416d01fd9https://github.com/elastic/security-docs/pull/6853/commits/6f04b4879e28d8bcb75ff0dad7a0f8f416d01fd9.

> * Improves the Dashboards overview page by moving visualization responses to visualization tables ({kibana-pull}214888[#214888]).

Author: ```suggestion
* Fixes a bug that caused the Dashboard overview page to crash when a Lens widget aggregation error occurred ({kibana-pull}214888[#214888]).
```

> * Improves the dashboards overview page by moving visualization responses to visualization tables ({kibana-pull}214888[#214888]).

Reviewer: ```suggestion
* Improves the Dashboards overview page by moving visualization responses to visualization tables ({kibana-pull}214888[#214888]).
```

> * Fixes a bug that caused the Dashboard overview page to crash when a Lens widget aggregation error occurred ({kibana-pull}214888[#214888]).

Reviewer 3: ```suggestion
* Fixes a bug that caused the Dashboard overview page to crash when a Lens widget aggregation error occurred ({kibana-pull}214888[#214888]).
* Fixes a bug in {elastic-defend} 8.16.0 where {elastic-endpoint} would incorrectly report some files as being `.NET`.
```
